.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+<!-- 
+---------------------------------------------------
+GENERAL SUPPORT INFORMATION
+---------------------------------------------------
+
+The GitHub issue tracker is for bug reports and feature requests.
+General support can be found at the following locations:
```

Comment:
You should probably put the IRC link at the top because people tend to have a problem with reading.